src/h3lib/lib/localij.c

```diff
@@ -519,10 +519,15 @@ H3Error localIjkToCell(H3Index origin, const CoordIJK *ijk, H3Index *out) {
  *
  * @param origin An anchoring index for the ij coordinate system.
  * @param index Index to find the coordinates of
+ * @param flags Mode flags, must be 0
```

Comment:
Do we want this to be a set of bitflags or an integer mode? I thought we went for the latter because we couldn't think of any combinatorial flags that would make any sense?

Reply:
I'm treating it as an opaque mode identifier without structure until we decide further.

Reply:
So, I meant the docs all say "flags"? And I thought we had decided to just make it a mode enum/integer?

Reply:
I see what you mean. In my mind these were interchangeable (I could just say there's a single flags bit field of length 32 bits). I standardized on `mode`.